backend/src/main/java/org/hackbrooklyn/plaza/security/Roles.java

```diff
@@ -44,39 +49,50 @@
                 Authorities.SAVED_EVENTS_ADD,
                 Authorities.SAVED_EVENTS_REMOVE,
                 Authorities.SAVED_EVENTS_READ,
-        };
-
-        // Volunteers
-        String[] volunteerAuthorities = {
-                // Announcements
-                Authorities.ANNOUNCEMENTS_READ,
 
-                // Events
-                Authorities.EVENTS_READ,
+                // Users
+                Authorities.USERS_CREATE,
+                Authorities.USERS_UPDATE_ROLE
+        );
 
-                // Saved Events
-                Authorities.SAVED_EVENTS_ADD,
-                Authorities.SAVED_EVENTS_REMOVE,
-                Authorities.SAVED_EVENTS_READ,
-        };
+        // Volunteers
+        List<String> volunteerAuthorities = Arrays.asList(
+                // TODO: Add volunteer authorities
+        );
 
         // Participants
-        String[] participantAuthorities = {
+        List<String> participantAuthorities = Arrays.asList(
                 // Announcements
-                Authorities.ANNOUNCEMENTS_READ,
+                Authorities.ANNOUNCEMENTS_READ_PARTICIPANTS_ONLY,
```

Comment:
Would Participants not be able to see the public announcements once they are signed in?

Reply:
Participants would in fact be able to see public announcements. Later on in the code, we combine the authorities for applicants (with public announcement access) with the authorities for participants, granting participants access to public announcements.